package/toltec-bootstrap/toltecctl

```diff
@@ -169,48 +313,72 @@ uninstall-force() {
     reboot
 }
 
-uninstall() {
-    # Remove all installed packages after prompting user
-    if [[ $# -gt 0 && ($1 = "-f" || $1 = "--force") ]]; then
-        uninstall-force
-        return
-    fi
-
-    read -p "This will wipe out all files in '$toltec_dest'. Continue? [y/n] " -n 1 -r
-    echo
-
-    if [[ $REPLY =~ ^[Yy]$ ]]; then
-        uninstall-force
-        return
-    fi
-
-    echo "Canceled"
-}
-
 help() {
     echo "Usage: $(basename "$0") COMMAND
 Manage your Toltec install. Available commands:
 
-    reenable        Re-enable Toltec after a system update.
-    set-path        Fix PATH value if needed.
-    uninstall       Permanently remove Toltec."
+    generate-opkg-conf      Rebuild the Opkg configuration file.
+    switch-branch [BRANCH]  Change the current branch to BRANCH.
+    reenable                Re-enable Toltec after a system update.
+    uninstall               Permanently remove Toltec."
 }
 
-if [[ $# -eq 0 ]]; then
-    help
-    exit 1
-fi
-
-action="$1"
-shift
-
-case $action in
-    reenable) reenable "$@" ;;
-    set-path) set-path "$@" ;;
-    uninstall) uninstall "$@" ;;
-    *)
-        echo -e "Error: Invalid command '$action'\n"
+if [[ $0 = "${BASH_SOURCE[0]}" ]]; then
+    if [[ $# -eq 0 ]]; then
         help
         exit 1
-        ;;
-esac
+    fi
+
+    action="$1"
+    shift
+
+    case $action in
+        generate-opkg-conf)
+            generate-opkg-conf && opkg update
```

Comment:
```suggestion
            generate-opkg-conf || return
            read -p "The config was updated. Do you want to apply it now ($ opkg update)? [Y/n]" -n 1 -r
            echo
            
            if [[ ! $REPLY =~ ^[Nn]$ ]]; then
                echo "Not updating for now."
                return
            fi
            
            opkg update
```

I'm not sure if `toltecctl generate-opkg-conf` should really do an update without asking (a user might want to verify the new config before it updates). Would be nice to add a prompt like suggested. Maybe even have some kind of `--no-update` flag.

Reply:
Does this also apply to `toltecctl switch-branch`?

Reply:
(I’m assuming it does.)